src/ReadableAttributes.php

```diff
@@ -4,6 +4,9 @@
 
 trait ReadableAttributes
 {
+    /** @var Container */
+    private $attributes;
```

Comment:
OK, I will change to `protected`.

The `phan` told me `undeclared property`, so I would like to modify this.

```
% phan -l src/                                                                                                                                                        
src/ReadWriteAttributes.php:32 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:9 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:19 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:29 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:39 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:49 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:59 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:69 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:79 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:89 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:99 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:109 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:119 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:129 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:139 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:149 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:159 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:169 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:179 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:189 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
src/ReadableAttributes.php:199 PhanUndeclaredProperty Reference to undeclared property \TurmericSpice\ReadableAttributes->attributes
```